scripts/uberenv/gen_spack_env_script.py

```diff
@@ -0,0 +1,128 @@
+###############################################################################
+# Copyright (c) 2015-2019, Lawrence Livermore National Security, LLC.
+#
+# Produced at the Lawrence Livermore National Laboratory
+#
+# LLNL-CODE-716457
+#
+# All rights reserved.
+#
+# This file is part of Ascent.
+#
+# For details, see: http://ascent.readthedocs.io/.
```

Comment:
Does this need to be updated?

Reply:
I took these directly from the uberenv repo ((https://github.com/LLNL/uberenv) and didn't make any changes so I'd prefer not to touch them at all.